plugins/cpp_metrics/model/include/model/cppastnodemetrics.h

```diff
@@ -0,0 +1,36 @@
+#ifndef CC_MODEL_CPPFUNCTIONMETRICS_H
+#define CC_MODEL_CPPFUNCTIONMETRICS_H
+
+#include <../../../../plugins/cpp/model/include/model/cppastnode.h>
```

Comment:
`${cpp_PLUGIN_DIR}/model/include` should be added to the include path instead and use:
```cpp
#include <model/cppastnode.h>
```

Reply:
Unfortunately, this solution does not work.

Reply:
Initially fixed in 8fca9f6754a68d4ba53416c6637647fcdeb309f4, as we discussed.

Reply:
Robust solution delivered in a84857cb192b999a95ef23aebc907a160465061e.